training/cloud/Containerfile

```
@@ -0,0 +1,8 @@
+ARG BOOTC_IMAGE=registry.stage.redhat.io/rhelai1/bootc-nvidia-rhel9:latest
```

The other Containerfiles have a generic image name here. Putting Red Hat is kind of limiting.
```suggestion
ARG BOOTC_IMAGE=quay.io/ai-labs/bootc-nvidia:latest
```

---

I wanted to use the official RHEL AI image as a sane default, to make it easier for customers that need to come here and run it

At least for now, until we're delivering official images for all the clouds

---

I finally did as you suggest, and changed a bit the variables and variable names to be more coherent with the rest of the repo